### PR TITLE
universal-ctags: init at 2016-07-06

### PR DESCRIPTION
**Pull request by mimame**
###### Motivation for this change

Add universal-ctags to NixOS
###### Things done
- [X] Tested using sandboxing
  ([nix.useChroot](http://nixos.org/nixos/manual/options.html#opt-nix.useChroot) on NixOS,
    or option `build-use-chroot` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file)
    on non-NixOS)
- Built on platform(s)
  - [X] NixOS
  - [ ] OS X
  - [ ] Linux
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nox --run "nox-review wip"`
- [X] Tested execution of all binary files (usually in `./result/bin/`)
- [X] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

---

### CODE REVIEW

File: pkgs/development/tools/misc/universal-ctags/default.nix [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+  postConfigure = ''
+    sed -i 's|/usr/bin/env perl|${perl}/bin/perl|' misc/optlib2c
+  '';
+
+  meta = with stdenv.lib; {
+    description = "A maintained ctags implementation";
+    homepage = "https://ctags.io/";
+    license = licenses.gpl2Plus;
+    platforms = platforms.unix;
+    # universal-ctags is preferred over emacs's ctags
```
Comment (jagajaga) [inline pkgs/development/tools/misc/universal-ctags/default.nix]: Why is this needed?

Comment (mimadrid) [inline pkgs/development/tools/misc/universal-ctags/default.nix]: For linking this ctag version over the emacs ctags version if emacs is also installed. See https://github.com/NixOS/nixpkgs/blob/385b21c92d6067e5f076ff29de54cf4d191d3ee5/pkgs/development/tools/misc/ctags/default.nix#L32
